api/src/pcapi/connectors/dms/serializer.py

```diff
@@ -75,13 +83,28 @@ def parse_beneficiary_information_graphql(
                 )
                 logger.error("Could not parse birth date %s for DMS application %s", value, application_number)
 
-        elif label in (dms_models.FieldLabel.TELEPHONE_FR.value, dms_models.FieldLabel.TELEPHONE_ET.value):
+        elif dms_models.FieldLabelKeyword.ID_PIECE_NUMBER.value in label:
+            value = value.strip()
+            if not fraud_api.validate_id_piece_number_format_fraud_item(value):
+                field_errors.append(
+                    dms_types.DmsFieldErrorDetails(key=dms_types.DmsFieldErrorKeyEnum.id_piece_number, value=value)
+                )
+            else:
+                id_piece_number = value
+
+        elif dms_models.FieldLabelKeyword.TELEPHONE.value in label:
             phone = value.replace(" ", "")
-        elif label in (
-            dms_models.FieldLabel.POSTAL_CODE_ET.value,
-            dms_models.FieldLabel.POSTAL_CODE_FR.value,
-            dms_models.FieldLabel.POSTAL_CODE_OLD.value,
-        ):
+
+        # TL;DR: The order between POSTAL_CODE and CITY check is important. Postal code needs to happen first.
+        #
+        # This is because the current labels are the following (as of 2022-07-05):
+        #
+        # POSTAL_CODE: "Quel est le code postal de ta commune de résidence ?"
+        # CITY:        "Quelle est ta commune de résidence ?"
+        #
+        #   If we check CITY first, we will get a false positive for the postal code,
+        #   because both labels match the 'commune de résidence' keyword
```

Comment:
Top merci. Je dirais qu'on peut simplifier en : 

```python
# The order between POSTAL_CODE and CITY check is important. Postal code needs to happen first.
# If we check CITY first, we will get a false positive for the postal code,
# because both labels match the 'commune de résidence' keyword
```

Avec le rappel du label ("commune de résidence") on comprend tout de suite le problème.
Qu'en penses-tu ?